packages/wonder-blocks-dropdown/src/components/__tests__/multi-select.test.js

```diff
@@ -379,9 +379,6 @@ describe("MultiSelect", () => {
             .find(OptionItem)
             .at(2)
             .find(ClickableBehavior);
-        // The focus is on opener. Press up (or down) should focus the input
-        select.simulate("keydown", {keyCode: keyCodes.up});
-        select.simulate("keyup", {keyCode: keyCodes.up});
```

Comment:
Oh this makes total sense! I'm glad that the browser now handles focus correctly without having to specify that.